.github/workflows/release-notes.yml

```diff
@@ -0,0 +1,24 @@
+name: Release Notes
+
+on:
+  push:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        compiler: [clang]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Environment setup
+      uses: actions/checkout@v2
+    - name: Check release notes
+      env:
+        CC: ${{ matrix.compiler }}
+        FC: gfortran-9
```

Comment:
I don't know if it saves time, but we don't need a Fortran compiler (or any kind of compiler) for this check. Just Git.

Reply:
Good point - fixed